src/uu/head/src/head.rs

```diff
@@ -1,240 +1,556 @@
-//  * This file is part of the uutils coreutils package.
-//  *
-//  * (c) Alan Andrade <alan.andradec@gmail.com>
-//  *
-//  * For the full copyright and license information, please view the LICENSE
-//  * file that was distributed with this source code.
-//  *
-//  * Synced with: https://raw.github.com/avsm/src/master/usr.bin/head/head.c
-
-#[macro_use]
-extern crate uucore;
-
-use std::collections::VecDeque;
-use std::fs::File;
-use std::io::{stdin, BufRead, BufReader, Read};
-use std::path::Path;
-use std::str::from_utf8;
-
-static SYNTAX: &str = "";
-static SUMMARY: &str = "";
-static LONG_HELP: &str = "";
-
-enum FilterMode {
-    Bytes(usize),
+use clap::{App, Arg};
+use std::convert::TryFrom;
+use std::ffi::OsString;
+use std::io::{ErrorKind, Read, Seek, SeekFrom, Write};
+use uucore::{crash, executable, show_error};
+
+const EXIT_FAILURE: i32 = 1;
+const EXIT_SUCCESS: i32 = 0;
+const BUF_SIZE: usize = 65536;
+
+mod options {
+    pub const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
please move usage, about and version out of "mod options" (like in the other programs)